bin/migrate-oats-data/applications/decisions/app_components/lot/application_decision_component_lot_insert.py

```diff
@@ -0,0 +1,143 @@
+from common import (
+    OATS_ETL_USER,
+    setup_and_get_logger,
+    BATCH_UPLOAD_SIZE
+)
+from db import inject_conn_pool
+from psycopg2.extras import RealDictCursor, execute_batch
+
+etl_name = "insert_application_component_lots"
+logger = setup_and_get_logger(etl_name)
+
+
+@inject_conn_pool
+def init_application_component_lots(conn=None, batch_size=BATCH_UPLOAD_SIZE):
+    """
+    This function is responsible for initializing the application_decision_component_lot in ALCS.
+
+    Args:
+    conn (psycopg2.extensions.connection): PostgreSQL database connection. Provided by the decorator.
+    batch_size (int): The number of items to process at once. Defaults to BATCH_UPLOAD_SIZE.
+    """
+
+    logger.info(f"Start {etl_name}")
+    with conn.cursor(cursor_factory=RealDictCursor) as cursor:
+        with open(
+            "applications/decisions/sql/components/application_decision_component_lot_insert_count.sql",
+            "r",
+            encoding="utf-8",
+        ) as sql_file:
+            count_query = sql_file.read()
+            cursor.execute(count_query)
+            count_total = dict(cursor.fetchone())["count"]
+        logger.info(f"Total Application Component Lots data to insert: {count_total}")
+
+        failed_inserts = 0
+        successful_inserts_count = 0
```

Comment:
suggestion: since both are counts they should both have _count. I see that this is copied and pasted a bunch only some of them are fixed, might want to check if theres others.

Reply:
current file has been updaed